Sensors.hpp

```diff
@@ -21,6 +25,12 @@ using namespace std;
 #elif TTGO_TQ
 #define PMS_RX 13  // config for TTGO_TQ board
 #define PMS_TX 18
+#elif ESP32DevKit
```

Comment:
These variables, PMS_RX and PMS_TX has not longer used in the current development. Sorry, I dont removed it yet. Its possible that the detection of Serial, serialInit() method try to pre-configured serial pines, if it not works, we should need write a example for each board.

Reply:
ok